tests/templates/test_layers/test_gate_fabric.py

```diff
@@ -764,10 +764,13 @@ def circuit_decomposed(weights):
     include_pi_param = qml.math.array(np.pi, like=qml.math._multi_dispatch(weights))
     for layer in range(weights.shape[0]):
         for idx in range(weights.shape[1]):
-            qml.OrbitalRotation.decomposition(include_pi_param, wires=qwires[idx])
-            qml.DoubleExcitation.decomposition(weights[layer][idx][0], wires=qwires[idx])
-            qml.OrbitalRotation.decomposition(weights[layer][idx][1], wires=qwires[idx])
-
+            with qml.tape.stop_recording():
+                op1 = qml.OrbitalRotation(include_pi_param, wires=qwires[idx])
```

Comment:
This is super ugly. Is there any way around this?

Reply:
I fear this is a very typical example for the weaknesses of the queuing system, and it does not only appear here, but is a general pattern! Every time we create an operation to call its method which creates other operations (i.e. to simplify an already created hamiltonian) this happens. 

But the alternative, to keep all methods static, is not a scalable option I fear.

I hope we can solve this elsewhere.